docs/fastrtpsgen/geninfo/geninfo.rst

```diff
@@ -31,18 +31,18 @@ It also provides an initial implementation of a publisher and a subscriber using
 Compile
 -------
 
-In order to compile *fastrtpsgen* you first need to have `gradle <https://gradle.org/install>`_ and
+In order to compile *Fast DDS-Gen* you first need to have `gradle <https://gradle.org/install>`_ and
 `java JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ installed
 (please, check the JDK recommended version for the gradle version you have installed).
 
-To compile *fastrtpsgen* java application, you will need to download its source code from
+To compile *Fast DDS-Gen* java application, you will need to download its source code from
 the `Fast-RPTS-Gen <https://github.com/eProsima/Fast-RTPS-Gen>`_ repository and with ``--recursive`` option and
 compile it calling ``gradle assemble``. For more details see :ref:`compile-fastrtpsgen`.
 
 .. code-block:: bash
 
     > git clone --recursive https://github.com/eProsima/Fast-RTPS-Gen.git
-    > cd Fast-RTPS-Gen
+    > cd Fast-DDS-Gen
```

Comment:
Should leave this as it was.
Cloning "Fast-RTPS-Gen" will still create a "Fast-RTPS-Gen" folder